resource-managers/kubernetes/core/pom.xml

```diff
@@ -108,6 +108,14 @@
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-servlet</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-http</artifactId>
+    </dependency>
```

Comment:
So if we didn't have this addition, would we still be able to build the whole module?  I don't see any new jetty-related code in this diff, so it seems like the pom change is independent from the exiting on apiserver unavailable change.

cc @mccheah